src/Components/Layouts/Pages/HomePage/NewsFeed/NewsVideo.js

```diff
@@ -28,17 +25,14 @@ export default class NewsVideo extends React.Component{
     render() {
         return (
             <div className={'uploadsContainer'}>
-            <Grid container xs={8} spacing={1}>
-                {this.state.profileNewsFeed && this.state.profileNewsFeed.map(function (upload) {
-                    return (
-                    <React.Fragment>
-                        <UserUploadsShort name={upload.name} surname={upload.surname} photo={upload.photo}/>
-                        <UploadedVideo url={upload.uploadLink} />
-                        <UploadedDesc description={upload.description} hashtags={upload.hashtags} />
-                    </React.Fragment>
-                    )
-                })}
-            </Grid>
+            {this.state.profileNewsFeed && this.state.profileNewsFeed.map((upload) => (
+                <div className={'uploadsSubContainer'}>
+                    <UserUploadsShort key={upload.id} name={upload.name} surname={upload.surname} photo={upload.photo}/>
```

Comment:
you should add key to parent div, not to all children. Doing it your way causes errors, because multiple elements have same key.